src/polymer/js-utils.ts

```diff
@@ -146,30 +146,26 @@ export function toScannedMethod(
 export function getOrInferPrivacy(
     name: string,
     annotation: jsdoc.Annotation|undefined,
-    privateUnlessDocumented: boolean): Privacy {
+    inferFromName = true,
```

Comment:
I'm not sure when class names would need to be private. If we're documenting classes that are not added to a namespace or registered as an element, maybe we should not do that.